install.sh

```diff
@@ -26,6 +26,10 @@ echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 # Configure dnsmasq
 ltsp-config dnsmasq
 
+# Client reboot issue fix
+echo 'INIT_COMMAND_RM_NBD_CHECKUPDATE="rm -rf /usr/share/ldm/rc.d/I01-nbd-checkupdate"' \
```

Comment:
Do not use `-r` if you're deleting file. I would suggest to move file to `I01-nbd-checkupdate.orig`  instead removing it.